lib/surface/components/link.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Surface.Components.Link do
+  @moduledoc """
+  Generates a hyperlink.
+
+  Provides a wrapper for Phoenix.HTML.Link's `link/2` function.
+
+  All options passed via `opts` will be sent to `link/2`, `label` and `class` can
+  be set directly and will override anything in `opts`.
+
+  ## Examples
+  ```
+  <Link
+    label="user"
+    to="/users/1"
+    class="is-danger"
+    opts={{ [method: :delete, data: [confirm: "Really?"]] }}
```

Comment:
If you rebase your code, you'll be able to pass the options without the outer brackets:

```
opts={{ method: :delete, data: [confirm: "Really?"] }}
```

Reply:
@msaraiva Rebased and squashed